pkg/errors/errors.go

```diff
@@ -47,6 +47,7 @@ var (
 	Unknown              error = errors.New("Unknown Reason")
 	DeleteFailed         error = errors.New("Delete Failed")
 	DeletePending        error = errors.New("Delete Pending")
+	WMIError             error = errors.New("WMI Error")
```

Comment:
Actually, we don't want to define WMIError here. We want to get the WMI error from the WMI module.